cmdx.py

```diff
@@ -4143,6 +4268,45 @@ def __exit__(self, exc_type, exc_value, tb):
 Context = DGContext
 
 
+def _encode_object(arg):
```

Comment:
These need some documentation. Why do they exist, what do they do?

Reply:
yeah this is more like my personal use of cmdx. feel free to drop it too

i found it useful to have these ones when i refactored scripts since they use a lot of maya commands and i had to constantly switch between strings and Node instances.